### PR TITLE
feat: disassemble php rule (933100 PL1)

### PR DESCRIPTION
**Pull request by franbuehler**
This PR disassembles the rule 933100 without changing its behaviour.
After merging this PR, I'll solve the issue #3616 and extend the regex.

I hope this two step approach makes the changes easier to understand.

### CODE REVIEW

File: regex-assembly/933100.ra [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    php
+  ##!<
+##!<
+
+##!> assemble
+  \[
+  ##!=>
+  ##!> assemble
+    php\]
+    \/php\]
```
Comment (theseion) [inline regex-assembly/933100.ra:23]:
```suggestion
    /php\]
```

**Review by theseion — CHANGES_REQUESTED** [review]

File: regex-assembly/933100.ra [hunk 2]
```diff
@@ -0,0 +1,26 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
```
Comment (theseion) [inline regex-assembly/933100.ra:3]: I'd probably through in an ignore case flag for good measure, PHP tags can be uppercase.

**Review by theseion — CHANGES_REQUESTED** [review]

File: regex-assembly/933100.ra (hunk 2 above)
Comment (franbuehler) [reply regex-assembly/933100.ra:3]: Thank you. Resolved.

**Comment by franbuehler** [comment]
Oh no! Now my lint test failed: t:lowercase and ignore case flag ;-)

**Review by theseion — APPROVED** [review]
Nice!

**Comment by franbuehler** [comment]
Thank you for reviewing and approving!

**Pull request merged**